cli/flox/src/main.rs

```diff
@@ -54,7 +54,7 @@ fn main() -> ExitCode {
 
     // Quit early if `--version` is present
     if Version::check() {
-        println!("Version: {}", *FLOX_VERSION);
+        println!(*FLOX_VERSION);
```

Comment:
```suggestion
        println!("{}", *FLOX_VERSION);
```

Reply:
Would `println!("{*FLOX_VERSION}");` also work for you, or is that discouraged syntax in this project?

Reply:
We do prefer "{var}" in general, but with the `*` that doesn't seem to compile:
```
error: invalid format string: expected `'}'`, found `'*'`
  --> flox/src/main.rs:57:20
   |
57 |         println!("{*FLOX_VERSION}");
   |                   -^ expected `'}'` in format string
   |                   |
   |                   because of this opening brace
   |
   = note: if you intended to print `{`, you can escape it using `{{`
```

Reply:
Oh, that's unexpected (and disappointing). I should have tried, obviously, but I'm so new to Rust that I do not even have a development environment installed yet 😬

Reply:
Yeah a bit disappointing after using Python's f-strings for me